src/ValueObjects/FacetResult.php

```diff
@@ -2,67 +2,49 @@
 
 namespace CultuurNet\SearchV3\ValueObjects;
 
-class FacetResult
+final class FacetResult
 {
-
     /**
      * Field where facets are based on.
      * @var string
      */
-    protected $field;
+    private $field;
 
     /**
-     * All results for this facet.
      * @var FacetResultItem[]
      */
-    protected $results;
+    private $results = [];
```

Comment:
This property gets overridden in the constructor.

Reply:
Fixed in https://github.com/cultuurnet/search-v3/pull/43/commits/9268fd6f232389803cf606f39342d6fe054878a3